README.md

```diff
@@ -20,13 +40,23 @@ rav1e is an AV1 video encoder. It is designed to eventually cover all use cases,
 * Constant quantizer and target bitrate (single- and multi-pass) encoding modes
 * Still picture mode
 
-## Releases
+## Documentation
+Find the documentation in [`doc/README.md`](doc/README.md)
+* AOM_ANALYZER
```

Comment:
What is this list about?

Reply:
This is table of contents of the documentation. I thought it is helpful for people to know what they can find there.

Reply:
if you make links to the files it might help a bit, as it is is quite confusing.

Reply:
If you think it is not helpful I will remove it to keep it compact.